atomicfu/src/jsAndWasmSharedMain/kotlin/kotlinx/atomicfu/locks/Synchronized.kt

```diff
@@ -2,9 +2,10 @@ package kotlinx.atomicfu.locks
 
 import kotlinx.atomicfu.REENTRANT_LOCK
 
+@Suppress("ACTUAL_CLASSIFIER_MUST_HAVE_THE_SAME_MEMBERS_AS_NON_FINAL_EXPECT_CLASSIFIER")
```

Comment:
```suggestion
@Suppress("ACTUAL_CLASSIFIER_MUST_HAVE_THE_SAME_MEMBERS_AS_NON_FINAL_EXPECT_CLASSIFIER_WARNING")
```

Reply:
But `kotlin.AllowDifferentMembersInActual` is not resolved for some reason, though dependency to `kotlin-stdlib-wasm-js` is provided.

Reply:
Just remove this suppress. `AllowDifferentMembersInActual` was removed for 1.9.20 for all targets.

Reply:
Oh, ok

Reply:
Well, `AllowDifferent....` was actually removed from 1.9.20, but without the supression the warning is still here:
```
w: file:///Users/Filipp.Zhinkin/Development/kotlinx-atomicfu/atomicfu/src/jsAndWasmSharedMain/kotlin/kotlinx/atomicfu/locks/Synchronized.kt:5:25 Actual typealias 'SynchronizedObject': actual class and its non-final expect class must declare exactly the same non-private members. The following non-private members in actual class are mismatched:

property '_hashCode': the visibility of this member must be the same in the expect class and the actual class. This error happens because the expect class 'SynchronizedObject' is non-final

property 'typeInfo': the visibility of this member must be the same in the expect class and the actual class. This error happens because the expect class 'SynchronizedObject' is non-final
```

Reply:
* and the ID of the warning seems to have changed since pre-release 1.9.20 builds, that's why I proposed the fix above.

Reply:
Ok, thank you! I just didn't see the warning is still there